.github/vendor/github.com/grafana/loki-release/workflows/release.libsonnet

```diff
@@ -161,12 +161,12 @@ local pullRequestFooter = 'Merging this PR will release the [artifacts](https://
         common.fetchReleaseLib,
         common.googleAuth,
         common.setupGoogleCloudSdk,
-        step.new('Set up QEMU', 'docker/setup-qemu-action@v3'),
+        step.new('Set up QEMU', 'docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392'), # v3295f81e9208d7d86ff1c6c12d2833863392'), # v3
```

Comment:
These need to be fixed in the `grafana/loki-release` repo https://github.com/grafana/loki-release/blob/main/workflows/release.libsonnet